AnkiDroid/src/main/java/com/ichi2/anki/NoteEditor.java

```diff
@@ -1631,6 +1642,11 @@ private boolean updateField(FieldEditText field) {
     }
 
 
+    private String convertToHtmlNewline(String fieldData) {
```

Comment:
```suggestion
    private String convertToHtmlNewline(@NonNull String fieldData) {
```

Maybe? (I think that's the right annotation - it's being called with no null check, so seems safer)

Reply:
Got to be - would throw otherwise